### PR TITLE
REF: PSBTWithHardware screen uses Hooks

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/j1zwyn7vxz6m0a8rwa0rkb0yww

[download apk](http://filestorage.bluewallet.io:1488/1605836507.apk)

File: screen/send/psbtWithHardwareWallet.js [hunk 1]
```diff
@@ -16,261 +15,169 @@ import {
...
-          <BlueText style={styles.hexLabel}>{loc.send.create_this_is_hex}</BlueText>
-          <TextInput style={styles.hexInput} height={112} multiline editable value={this.state.txhex} />
+      <View style={[styles.rootPadding, stylesHook.rootPadding]}>
+        <BlueCard style={[styles.hexWrap, stylesHook.hexWrap]}>
+          <BlueText style={[styles.hexLabel, stylesHook.hexLabel]}>{loc.send.create_this_is_hex}</BlueText>
+          <TextInput style={[styles.hexInput, stylesHook.hexInput]} height={112} multiline editable value={txHex} />
 
-          <TouchableOpacity style={styles.hexTouch} onPress={() => Clipboard.setString(this.state.txhex)}>
-            <Text style={styles.hexText}>{loc.send.create_copy}</Text>
+          <TouchableOpacity style={styles.hexTouch} onPress={Clipboard.setString}>
```
Comment (limpbrains) [inline screen/send/psbtWithHardwareWallet.js:157]: Do you need to pass `txHex` to `Clipboard.setString()` ?

File: screen/send/psbtWithHardwareWallet.js [hunk 2]
```diff
@@ -338,96 +244,148 @@ export default class PsbtWithHardwareWallet extends Component {
...
+    backgroundColor: 'rgba(0,0,0,0.4)',
+    justifyContent: 'center',
+    borderRadius: 20,
+    position: 'absolute',
+    right: 16,
+    top: 64,
+  },
+  closeCameraImage: {
+    alignSelf: 'center',
+  },
```
Comment (limpbrains) [inline screen/send/psbtWithHardwareWallet.js:358]: Some styles are unused, like rootCamera, closeCameraImage

**Review by limpbrains — CHANGES_REQUESTED** [review]

File: screen/send/psbtWithHardwareWallet.js (hunk 2 above)
Comment (marcosrdz) [reply screen/send/psbtWithHardwareWallet.js]: Fixed

File: screen/send/psbtWithHardwareWallet.js (hunk 1 above)
Comment (marcosrdz) [reply screen/send/psbtWithHardwareWallet.js]: Fixed

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/zc03z8dw0ekpzkarahjcda8z6g

[download apk](http://filestorage.bluewallet.io:1488/1605880059.apk)

**Review by limpbrains — APPROVED** [review]

**Comment by Overtorment** [comment]
lets run all tests with fresh master

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/zqh5fd0w5mddy8hcurmxg3t360

[download apk](http://filestorage.bluewallet.io:1488/1605885385.apk)

File: Navigation.js [hunk 3]
```diff
@@ -146,7 +146,7 @@ const WalletsRoot = () => (
       component={LNDViewInvoice}
       options={LNDViewInvoice.navigationOptions}
       swipeEnabled={false}
-      gesturesEnabled={false}
+      gestureEnabled={false}
```
Comment (Overtorment) [inline Navigation.js:149]: typo fix?

**Review by Overtorment — APPROVED** [review]
lgtm. I hope we have enought e2e tests for this screen to catch if smth is principally broken

File: Navigation.js (hunk 3 above)
Comment (marcosrdz) [reply Navigation.js:149]: Nav 5 changed from gestures to gesture

**Pull request merged**